### PR TITLE
Feat/bs 13534 la usage tracking

### PR DESCRIPTION
**Pull request by abirembaut**
BS-13534 - Living application usage Tracking (lead generation)

### CODE REVIEW

File: common/src/main/java/org/bonitasoft/console/common/server/preferences/properties/PlatformPreferencesProperties.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package org.bonitasoft.console.common.server.preferences.properties;
```
Comment (apupier) [inline common/src/main/java/org/bonitasoft/console/common/server/preferences/properties/PlatformPreferencesProperties.java]: missing header license file

File: server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+/**
+ * @author Baptiste Mesta
+ * @author Aurelien Pupier
+ * @author Anthony Birembaut
+ */
+public class BonitaRegistration {
+
+    public static final String SERVICE_URL = "http://stats.bonitasoft.org/stats.php";
+    public static final String BONITA_REGISTER_SYSTEM_PROPERTY = "bonita.web.register";
+	public static final String BONITA_USER_REGISTER_TRY = "user.register.try";
```
Comment (apupier) [inline server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java]: indentation not correct, using tab here

File: server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        infos.put("bonita.version", new BonitaVersion(new VersionFile()).getVersion());
+        for (final String systemPropertyToSend : SYSTEM_PROPERTIES_TO_SEND) {
+            addSystemPropertyToMap(infos, systemPropertyToSend);
+        }
+        infos.put("field_user_systeme_lang", Locale.getDefault().toString());
+        infos.put("proc.number", String.valueOf(Runtime.getRuntime().availableProcessors()));
+        infos.put("mem.total", String.valueOf(Runtime.getRuntime().totalMemory() / 1048576) + "mo");
+        infos.put("mem.max", String.valueOf(Runtime.getRuntime().maxMemory() / 1048576) + "mo");
+        infos.put("origin", "business-app");
+		addIpAdressInfo(infos);
```
Comment (apupier) [inline server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java]: indentation with tab

File: server/src/test/java/org/bonitasoft/console/common/server/registration/BonitaRegistrationTest.java [hunk 4]
```diff
@@ -0,0 +1,123 @@
+package org.bonitasoft.console.common.server.registration;
```
Comment (apupier) [inline server/src/test/java/org/bonitasoft/console/common/server/registration/BonitaRegistrationTest.java]: Missing header

File: server/src/test/java/org/bonitasoft/console/common/server/registration/SystemInfoSenderTest.java [hunk 5]
```diff
@@ -0,0 +1,72 @@
+package org.bonitasoft.console.common.server.registration;
```
Comment (apupier) [inline server/src/test/java/org/bonitasoft/console/common/server/registration/SystemInfoSenderTest.java]: missing header

File: common/src/main/java/org/bonitasoft/console/common/server/preferences/properties/PlatformPreferencesProperties.java (hunk 1 above)
Comment (abirembaut) [inline common/src/main/java/org/bonitasoft/console/common/server/preferences/properties/PlatformPreferencesProperties.java]: fixed

File: server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java (hunk 2 above)
Comment (abirembaut) [inline server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java]: fixed

File: server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java (hunk 3 above)
Comment (abirembaut) [inline server/src/main/java/org/bonitasoft/console/common/server/registration/BonitaRegistration.java]: fixed

File: server/src/test/java/org/bonitasoft/console/common/server/registration/SystemInfoSenderTest.java (hunk 5 above)
Comment (abirembaut) [inline server/src/test/java/org/bonitasoft/console/common/server/registration/SystemInfoSenderTest.java]: fixed

File: server/src/test/java/org/bonitasoft/console/common/server/registration/BonitaRegistrationTest.java (hunk 4 above)
Comment (abirembaut) [inline server/src/test/java/org/bonitasoft/console/common/server/registration/BonitaRegistrationTest.java]: fixed

**Pull request merged**